### DISCUSSION TITLE
Set amount of chars per tab

### DISCUSSION THREAD

**Discussion by mindrones** [community/community#4752] [Mobile] [answered]
It would be quite useful being able to set the amount of chars displayed for each tab (`\t`).

On the mobile app this seems to be 6 chars which makes reading tabbed code difficult in portrait mode as the majority of the code ends up being offscreen.

To set this up there could be a dropdown or a numerical input under `Code options`.

This would be quite handy in the desktop version too by the way.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @mindrones – I've opened an issue for our team to discuss this and see what's possible. Agreed that large tab widths can make it harder to read code, so this would be a nice fix 😊

**Reply by istudyatuni** [reply]
Hi, could you check to see if there is any progress on this issue?

**Reply by brianlovin** [reply]
This hasn't landed on our roadmap yet, so it's not prioritized.

**Reply by istudyatuni** [reply]
Sadly. Thank you!
